components/Tabs/BUILD

```diff
@@ -173,18 +179,37 @@ mdc_objc_library(
     ],
 )
 
+mdc_objc_library(
+    name = "unit_test_tabbarview_sources",
+    testonly = 1,
+    srcs = native.glob([
+        "tests/unit/TabBarView/*.m",
+    ]),
+    sdk_frameworks = [
+        "XCTest",
+    ],
+    deps = [
+        ":TabBarView",
+    ],
+)
+
 mdc_unit_test_suite(
     name = "unit_tests",
     deps = [
         ":unit_test_sources",
+        ":unit_test_tabbarview_sources",
         ":unit_test_theming_sources",
     ],
 )
 
 mdc_snapshot_objc_library(
     name = "snapshot_test_lib",
-    extra_srcs = native.glob(["tests/snapshot/Theming/*.m"]),
+    extra_srcs = native.glob([
```

Comment:
Why the `extra_srcs` and not just `deps`? What is the difference, I'm just asking for my understanding, no action required.

Reply:
I'm not sure I understand - do you want a separate `objc_library` for the Extension theming sources and have the main Snapshot library target depend on that?

Reply:
This is no longer modified by this PR so I'm going to resolve this conversation. However, let's continue the discussion offline so we make sure we both understand what's going on here.

Reply:
Just looked at it too fast.